S26-documentation/07a-tables.t

```diff
@@ -16,11 +16,10 @@ my $r;
     =end table
     $r = $=pod[0];
     # 3 tests
-    is $r.contents.elems, 1;
-    is $r.contents[0][0], "-r0c0"; # <= note leading hyphen which caused the original issue
-    is $r.contents[0][1], "r0c1";
+    is $r.contents.elems, 1, "test {++$t}";
```

Comment:
I think the number is included in the test output anyway, so what are these descriptions for?

Reply:
The second two lines removed were redundant.  The "test {++$t}" was left from some debugging code--it can be removed.

Reply:
Remove it then? :)